client/landing/jetpack-cloud/sections/backups/index.js

```diff
@@ -16,29 +16,22 @@ import {
 	backupDownload,
 	backupRestore,
 } from 'landing/jetpack-cloud/sections/backups/controller';
+import { backupMainPath, backupRestorePath, backupDownloadPath, backupDetailPath } from './paths';
 
 export default function() {
 	if ( config.isEnabled( 'jetpack-cloud/backups' ) ) {
-		page( '/backups', siteSelection, sites, navigation, makeLayout, clientRender );
-		page( '/backups/:site', siteSelection, navigation, backups, makeLayout, clientRender );
```

Comment:
We should keep these routes to manage the cases without site slug or with a wrong site slug:

![image](https://user-images.githubusercontent.com/9832440/78254304-a145d180-74ed-11ea-8891-91ea92f9a1f9.png)

![image](https://user-images.githubusercontent.com/9832440/78255100-c5ee7900-74ee-11ea-94f2-50e322736895.png)

The same for the rest of the pages.

The previous behavior was to force the user to select a valid site from the list.

Reply:
they [have been moved to the bottom of the file.](https://github.com/Automattic/wp-calypso/pull/40630/files#diff-084f09d5b900e0e16c94f7793eaa408fR54)

This is actually not really required for this PR, but the `backups/:site` route frustrated me in #40494 where `backups/activity` was getting matched to it instead of its own handler.

HOWEVER! You did catch that my ternary for `backupMainPath` was backwards:
```js
siteName ?  '/backups  : `/backups/${ siteName }`;
``` 
instead of 
```js
siteName ? `/backups/${ siteName }` : '/backups';
```

🤦 everything should work as expected now!

Reply:
It works as expected